apiary.apib

```diff
@@ -676,6 +677,10 @@ Get a list of departments.
     + Attributes (object)
         + filter (object)
             + ids: `92296ad0-2d61-4179-b174-9f354ca2157f`,`53635682-c382-4fbf-9fd9-9506ca4fbcdd` (array[string], optional)
+            + status (array[enum], optional) - Filters on status
```

Comment:
It should be specified what the default behavior of this filter is (`active` if I understand correctly)

There should be (as I believe @mrnnn mentioned) a way to query active+archived together.

Reply:
I'm not sure what you mean with default behaviour of a filter, it's either filled in or not. Did you maybe mean the default behaviour of the .list endpoint itself?


Since this property is an array the behaviour of querying active & archived together is already available.

Reply:
If it's not filled in the default is `[active]` not `[active, archived]` as you'd expect from this specification.

Reply:
Right, I didn't think about the empty array possibility. 👍

Reply:
I think we don't need to describe this behaviour though, looking at the other other endpoints we never describe any behaviour, as I do slightly seem to remember this documentation is about possible request & response properties and not about documenting behaviour on/with those properties.

Although do you have a proposal we could maybe phrase this?

Reply:
Something like:

```
(default: `[active]`)
```

would be plenty I think?

Reply:
This remark became obsolete after always returning active & archived entities by default